services/api/src/messaging/players.service.ts

```diff
@@ -80,6 +120,12 @@ const setDefaultStatusForPlayers = (ws: GameWebSocket): void => {
 
 export const resetGame = (ws: GameWebSocket): void => {
   gameService.clearBoard(ws.roomId);
+  if (gameService.getVoters(ws.roomId).length < 2) {
```

Comment:
why is this condition needed?

Reply:
Because, without this condition, if one player joins the room and presses restart game button, the game state would become GameInProgress and he would be able to place figures on the board (at least in FE)